docs/FAQ.md

```diff
@@ -0,0 +1,177 @@
+<!-- omit in toc -->
+## Table of Contents
+- [Rationale FAQ](#rationale-faq)
+  - [What were the problems with the \<2023 Go IPFS repo set?  Why take action now? What's changed from 2022 and earlier?](#what-were-the-problems-with-the-2023-go-ipfs-repo-set--why-take-action-now-whats-changed-from-2022-and-earlier)
+  - [Are there other reaons/goals for doing repo consolidation?](#are-there-other-reaonsgoals-for-doing-repo-consolidation)
+  - [Why is the Boxo project consolidating repos and using example-driven development despite ecosystem friction?](#why-is-the-boxo-project-consolidating-repos-and-using-example-driven-development-despite-ecosystem-friction)
+  - [What are the risks in repo consolidation?](#what-are-the-risks-in-repo-consolidation)
+  - [How much does a repo cost?](#how-much-does-a-repo-cost)
+  - [What alternatives were considered and why were they ruled against?](#what-alternatives-were-considered-and-why-were-they-ruled-against)
+  - [Given this isn't free for maintainers or those affected, what has been the cost/benefit analysis?](#given-this-isnt-free-for-maintainers-or-those-affected-what-has-been-the-costbenefit-analysis)
+  - [What are some concrete usecases where repo consolidation would have been beneficial?](#what-are-some-concrete-usecases-where-repo-consolidation-would-have-been-beneficial)
+  - [Is this really for developers or just maintaienrs?](#is-this-really-for-developers-or-just-maintaienrs)
+  - [What kind of user outreach was done about this consolidation work?](#what-kind-of-user-outreach-was-done-about-this-consolidation-work)
+- [Copied or Migrated Repos FAQ](#copied-or-migrated-repos-faq)
+  - [What repos have been copied into Boxo?](#what-repos-have-been-copied-into-boxo)
+  - [What will happen if there is a security issue in one of the legacy ipfs/\* repos that has been copied into boxo?](#what-will-happen-if-there-is-a-security-issue-in-one-of-the-legacy-ipfs-repos-that-has-been-copied-into-boxo)
+  - [How does Boxo make sure it doesn't miss any important changes that are in the original/source/copied repos?](#how-does-boxo-make-sure-it-doesnt-miss-any-important-changes-that-are-in-the-originalsourcecopied-repos)
+  - [I don't like the "deprecated" warnings in "not maintained" repos that have been moved to boxo.  What can be done about this?](#i-dont-like-the-deprecated-warnings-in-not-maintained-repos-that-have-been-moved-to-boxo--what-can-be-done-about-this)
+  - [How does one claim ownership of a "not maintained" repo?](#how-does-one-claim-ownership-of-a-not-maintained-repo)
+  - [Will the "not maintained" ipfs/\* repos be left around to rot?](#will-the-not-maintained-ipfs-repos-be-left-around-to-rot)
+
+## Rationale FAQ
```

Comment:
This seems weird to call this an "FAQ", I don't think we're really going to maintain this as an FAQ based on what users frequently ask? We don't have any evidence that anyone outside of PL cares about these questions, and also it seems more like just some historical context around why Boxo exists and why a bunch of repos got moved into it. Instinctively this stuff feels more appropriate for a GH issue that we link to in the FAQ/README if someone wants to know the decision making that led to Boxo.

Reply:
That's good feedback Gus.  Stepping back it makes me think we should lean into Github wikis more for this kind of long form documentation.  It has better discoverability and navigation properties than issues and doesn't clog up the issue tracker.  (@Jorropo has floated this idea recently.). I'll do this now.